src/k8s/request.js

```diff
@@ -185,17 +185,24 @@ export const createVm = async (
   vmSettings,
   networks,
   storages,
-  persistentVolumeClaims
+  persistentVolumeClaims,
+  units
 ) => {
   const getSetting = settingsValue.bind(undefined, vmSettings);
 
   let conversionPod;
 
   if (isVmwareProvider(vmSettings)) {
-    const importResult = await importVmwareVm(vmSettings, networks, storages, {
-      k8sCreate,
-      k8sPatch,
-    });
+    const importResult = await importVmwareVm(
+      vmSettings,
+      networks,
+      storages,
+      {
+        k8sCreate,
+        k8sPatch,
+      },
+      units
```

Comment:
can we pass this as a object for better enhancement. `{ units }` ?

I guess we should have begin with the object parameters to begin with (maybe TODO for refactoring?). 

Only this have to be left the same (because of future possibility of passing `EnhancedK8sMethods`):
```js
{
        k8sCreate,
        k8sPatch,
}
```

Reply:
done